### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by 12345lcr**
# Changes

- Add issue templates of bug-report, feature-request and free-form

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+### Knative (serving/eventing/client) version
+
+<!-- Remove all except the known affected versions of Knative running on the cluster on which you have detected the issue -->
+
+> Nightly
+> 0.12.x
+> 0.11.x
+> 0.10.x
+> 0.9.x
```
Comment (rhuss) [inline .github/ISSUE_TEMPLATE/bug-report.md:39]: I think you can update this list, starting with 0.21 (0.12 is already super old and not supported anymore)

**Review by rhuss — COMMENTED** [review]
looks good with some minor comment

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (12345lcr) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Got it, thanks!

**Review by rhuss — APPROVED** [review]
thanks ! Looks good to me

**Pull request merged**